man/oci-runtime-tool-generate.1.md

```diff
@@ -60,6 +60,14 @@ read the configuration from `config.json`.
   The *fileMode*, *uid*, *gid* are optional.
     *fileMode* is the file mode of the device file.
     *uid*/*gid* is the user/group id of the device file.
+  This option can be specified multiple times.
+
+**--device-remove**=*TYPE:MAJOR:MINOR:PATH[:OPTIONS...]*
```

Comment:
type:major:minor:path should be enough to identify the device. There is not need to parse the options.